app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -203,6 +186,73 @@ fun Setting(
     }
 }
 
+/**
+ * Modifier to get rounded corners on the ripple that appears when applying
+ * a clickable/toggleable/etc to a composable.
+ *
+ * @param modifier clickable/toggleable/etc modifier with an indication.
+ * @return An interactable [Modifier] with a rounded indication.
+ */
+private fun Modifier.roundedInteractable(modifier: Modifier = Modifier): Modifier =
+    this
+        .clip(RoundedCornerShape(8.dp))
+        .then(modifier)
+        .padding(8.dp)
+
+@Composable
+fun SwitchSetting(
+    label: String,
+    description: String,
+    checked: Boolean,
+    onValueChange: (Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
```

Comment:
Do we need to add an `enabled` parameter to all of these? I don't think we have a need for it yet.